contrib/devtools/update-translations.py

```diff
@@ -122,6 +123,13 @@ def escape_cdata(text):
     text = text.replace('"', '&quot;')
     return text
 
+def check_bitcoin_addr(text):
+    p = re.compile('^([1,3]|bc1)[a-zA-Z0-9]{33,39}$')
```

Comment:
This is too strict and wouldn't match an address hidden between words.

Reply:
Agreed. I want to avoid false positives here. What do you suggest the best way forward is? I could split on any spaces in the string then perform this check, but that might be rather expensive, although I'm not sure if performance is all that important in this file? If I don't set some sort of string length in the regex pattern, there are going to be a bunch of false positives.

Reply:
Hmm, could you elaborate what you mean by false positives? Imo a word that is longer than 30 chars and includes at least one digit should never be part of a translation (regardless if it was a bitcoin address)

Reply:
You're right here. There were some translations that started with a 1 but they all contain spaces. The new commit I just pushed should work. Thanks for the input